xc/xenops_server_xen.ml

```diff
@@ -97,6 +97,7 @@ module VmExtra = struct
 		qemu_vifs: (Vif.id * (int * qemu_frontend)) list;
 		pci_msitranslate: bool;
 		pci_power_mgmt: bool;
+		pv_drivers_detected: bool;
```

Comment:
If we change `VmExtra`, we need to be very careful to handle the upgrade case. For example, when suspending a VM on an older release, VmExtra gets stored in the VM.last_booted_record in xapi. When resuming it, we need to make sure that a default gets filled in for new fields.